extensions/wikia/AdEngine/js/AdInfoTrackerHelper.js

```diff
@@ -67,6 +68,20 @@ define('ext.wikia.adEngine.adInfoTrackerHelper',  [
 		return data;
 	}
 
+	function transformBidderPrice(bidderName, realSlotPrices, slotPricesIgnoringTimeout) {
+		console.log(typeof realSlotPrices[bidderName], realSlotPrices[bidderName]);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this logging statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS2228)